docs/self-managed/platform-deployment/helm-kubernetes/guides/aws-marketplace.md

```diff
@@ -0,0 +1,494 @@
+---
+id: aws-marketplace
+title: "Install AWS Marketplace"
+description: "Learn how to Install the AWS Marketplace."
+---
+
+<!--- Can you give us a 2- or 3-sentence overview of the guide here and why a user would want to install this? --->
+
+Estimated time to deploy: 3 hours
+
+Required permissions:
+
+```
+eks:CreateCluster
+eks:DeleteCluster
+eks:DescribeCluster
+eks:ListClusters
+eks:TagResource
+eks:AccessKubernetesApi
+eks:ListNodegroups
+eks:CreateNodegroup
+eks:DescribeNodegroup
+eks:DeleteNodegroup
+eks:CreateAddon
+eks:ListAddons
+
+```
+
+## Community Helm profiles
+
+These instructions were based off of the [community Helm profiles](https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/main/aws/README.md) in GitHub.
+
+The workflow for these Helm profiles is to edit the top of these makefiles.
+
+<!--- Can you clarify for me here? Do you mean the purpose of these Helm profiles is to edit these files? --->
```

Comment:
Lastly, here 👍

Reply:
That section explained that I derived the instructions from the community-hub helm profiles.  I can probably remove that section, because it's not really relevant to customers.